Source/WebDriver/socket/HTTPServerSocket.cpp

```diff
@@ -108,7 +108,7 @@ void HTTPRequestHandler::sendResponse(HTTPRequestHandler::Response&& response)
 {
     auto& endpoint = RemoteInspectorSocketEndpoint::singleton();
     auto packet = packHTTPMessage(WTFMove(response));
-    endpoint.send(m_client.value(), packet.utf8().data(), packet.length());
+    endpoint.send(m_client.value(), reinterpret_cast<const uint8_t*>(packet.utf8().data()), packet.utf8().length());
```

Comment:
This looks like it would do the conversion twice, I suggest putting the `CString` in a temporary variable first.

Otherwise, the code change looks right to me.

Reply:
You are certainly correct. Thank you for pointing that out.
I fixed it with the commit https://github.com/WebKit/WebKit/commit/7c04ff82351442ac03bffa11cd1ba02bff27cd8b.